addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -449,13 +449,13 @@
 
                 <div t-if="o.currency_id != o.company_id.currency_id"
                      class="row clearfix ms-auto text-nowrap border-top border-bottom p-2">
-                    <t t-set="exchange_rate"
+                    <t t-if="o.amount_total" t-set="exchange_rate"
                        t-value="abs(o.amount_total_signed) / o.amount_total"/>
```

Comment:
```suggestion
                       t-value="abs(o.amount_total_signed) / o.amount_total if o.amount_total else 0"/>
```

Reply:
@jco-odoo @hupo-odoo is there a reason we compute the exchange rate here?

Reply:
yes, to show it in the report (The government wants to see the relation with the tax amount in company currency)

Reply:
> 

Hi, changes have been done, Can you please look into it?

Thanks